services/php-accountAndSignature/app/src/Domain/ValueObject/Password.php

```diff
@@ -0,0 +1,4 @@
+<?php
+namespace App\Domain\ValueObject;
+
+class Password extends StringValueObject { }
```

Comment:
### Feature Required
Password must be encrypted.
Feel free to implement encryption algorithm here in ValueObject or in Account Entity.